jobsdb/jobsdb.go

```diff
@@ -1332,13 +1472,13 @@ func (jd *HandleT) getUnprocessedJobsDS(ds dataSetT, customValFilters []string,
 	var sqlStatement string
 
 	if useJoinForUnprocessed {
-		sqlStatement = fmt.Sprintf(`SELECT %[1]s.job_id, %[1]s.uuid, %[1]s.parameters, %[1]s.custom_val,
+		sqlStatement = fmt.Sprintf(`SELECT %[1]s.job_id, %[1]s.uuid, %[1]s.user_id, %[1]s.parameters, %[1]s.custom_val,
```

Comment:
use sql.NullString for user_id to handle null user_id cases?

Reply:
Instead made user_id as NOT NULL and handling in the setup. Its required only for a release. And we can clean that up by the next release.